tests/test_tui_publish_workflow_security.py

```
@@ -1640,6 +1640,145 @@ def test_relay_attestations_survive_a_skipped_windows_build() -> None:
+    "cloudflare-relay",
+    "build-windows",
+    "package",
+    "verify-linux-packages",
+    "attest-npm-packages",
+)
+
+
+def _package_workflow_document() -> dict[str, object]:
+    document = yaml.load(
+        workflow("cmux-tui-build-package.yml"),
+        Loader=yaml.BaseLoader,
```

Ruff (0.16.3)

[error] Probable use of unsafe loader `BaseLoader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)